.travis.yml

```diff
@@ -1,27 +1,37 @@
 language: python
-# Workaround for Python 3.6 missing
-python: "3.6"
```

Comment:
This was causing all Travis builds to run on Python 3.6, which is probably not what we want... I've updated it to work as intended, its not as pretty but it works

Reply:
oh yeah. I've noticed that TravisCI has real support for Python 3.6 now